README.md

```diff
@@ -1,10 +1,23 @@
 # Blueprint Truffle Box
 
-This box comes with everything you need to create your own Truffle Box.
+This box comes with everything you need to get started on your own Truffle Box.
 
 ## Getting Started
 
-After adding your libraries and code:
+First, fork this repository to create your own repository and make edits. Then, start adding your libraries and code!
```

Comment:
Not actually sure if this the right process

Reply:
works for me 😊